docs/running-node-manual/on.md

```diff
@@ -0,0 +1,79 @@
+# Running an Observer Node manually
+
+## Prerequisites
+Make sure you have all [prerequisites](./prerequisites.md) set up
+## Deployment steps
+
+#### 1. Put `cosmovisor` binary to `/usr/bin/`, set proper owner and execution permissions.
+
+#### 2. Locate the genesis app version to genesis application version directory:
+- Create `$HOME/.dcl/cosmovisor/genesis/bin` directory.
+- Copy `dcld` binary to it, set proper owner and execution permissions.
+    Please note that execution permissions on `dcld` should be granted to all (i.e. User, Group and Others classes)
+    because cosmovisor requires execution permission on the application binary to be granted to Others class.
+
+#### 3. Configure CLI:
+- `./dcld config chain-id testnet`
+- `./dcld config output json` - Output format (text/json).
+
+#### 4. Initilize the node:
+
+```bash
+./dcld init "<node-name>" --chain-id "<chain-id>"
+```
+- Use `testnet` if you want to connect to the persistent Test Net
+
+#### 5. (Optional) Consider enabling `state sync` in the configuration if you are joining long-running network
+- For more information refer to [running-node-in-existing-network.md](../advanced/running-node-in-existing-network.md)
+
+#### *** Step 6 can be automated using `run_dcl_node` script:
+Run node:
+
+```bash
+./run_dcl_node -t observer -c "<chain-id>" "<node-name>"
+```
+
+> Notes:
+>
+> * the script assumes that:
+>   * current user is going to be used for `cosmovisor` service to run as
+>   * current user is in sudoers list
+
+#### 6. Run node:
+- Put `genesis.json` into dcld's config directory (usually `$HOME/.dcl/config/`).
+    - Use `deployment/persistent_chains/testnet/genesis.json` if you want to connect to the persistent Test Net
+- Open `$HOME/.dcl/config/config.toml` file in your favorite text editor:
+    - Tell node how to connect to the network:
+        - Set the value for the `persistent_peers` field as the content of `persistent_peers.txt` file.
+        - Use `deployment/persistent_chains/testnet/persistent_peers.txt` if you want to connect to the persistent Test Net.
```

Comment:
This is incorrect, the persistent peers for the Test Net should be obtained from a  Slack channel.